src/containers/Login/Greeting.tsx

```diff
@@ -28,11 +30,41 @@ const Picture = styled.img`
   height: 600px;
 `;
 
+const TabsWrapper = styled.div`
```

Comment:
would be better to move it to the components folder. perhaps, we will use this wrapper again) so, move it to components and rename it to tabContainer